database/infrastructure.py

```diff
@@ -13,17 +13,14 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+from aws_cdk import RemovalPolicy
```

Comment:
I'd prefer to keep module namespace. That should allow a reader to quicker understand which module `RemovalPolicy` belongs to:
```suggestion
import aws_cdk as cdk
```